doc/pages/architecture/090-database.md

```diff
@@ -0,0 +1,219 @@
+# Astarte Database
+
+Astarte might need to store a massive ammount of data, to accomplish this task a [Cassandra](http://cassandra.apache.org/) database is used. Cassandra offers scalability and high availability with [good performances](https://www.datastax.com/apache-cassandra-leads-nosql-benchmark).
+Cassandra offers linear scalability and can span from really small clusters to thousands of nodes, last but not least it provides a reliable storage for all devices data.
```

Comment:
`Cassandra offers linear scalability and can span from really small clusters to thousands of nodes, without compromising on reliability.`